Updating the llama index APIs to use the latest dependencies.

nchandrappa opened the pull request.

nchandrappa committed:
Updating the llama index APIs to use the latest dependencies.

nchandrappa committed:
Updating readme file.

ashetkar commented on README.md:
````diff
@@ -93,6 +91,7 @@ This application requires a database table with financial information for compan
 3. Execute the SQL files against the database.
     ```sh
+    docker exec -it yugabytedb-node1 bin/ysqlsh -h yugabytedb-node1 -f /home/schema_extended.sql
     docker exec -it yugabytedb-node1 bin/ysqlsh -h yugabytedb-node1 -f /home/schema.sql
````
It fails saying `ysqlsh:/Users/amoghshetkar/code/yugabytedb-llamaindex-sp500-search/sql/schema.sql:55: ERROR:  relation "companies" already exists`. The table will be created in `schema_extended.sql` in the earlier step.

nchandrappa committed:
Update README.md

Updating the readme to fix the docker container name used for copying the SQL files into the container.

nchandrappa committed:
Updating the table names which was missed in the original commit.

nchandrappa committed:
Fixing the table name in data_extended.sql

ashetkar commented on README.md:
```diff
@@ -23,16 +23,12 @@ Download the application and provide settings specific to your deployment:
         python3 -m venv yb-llamaindex-env
         source yb-llamaindex-env/bin/activate
         pip install -r requirements.txt
```
The step failed on centos with below error
```
ERROR: Ignored the following yanked versions: 2.4.2
ERROR: Could not find a version that satisfies the requirement aiohappyeyeballs==2.6.1 (from versions: 0.0.1, 0.1.0, 0.2.0, 0.3.0, 0.4.0, 0.5.0, 0.6.0, 0.7.0, 0.8.0, 0.9.0, 1.0.0, 1.1.0, 1.2.0, 1.3.0, 1.4.0, 1.4.1, 1.5.0, 1.6.0, 1.7.0, 1.8.0, 1.8.1, 2.0.0, 2.1.0, 2.2.0, 2.3.0, 2.3.1, 2.3.2, 2.3.3, 2.3.4, 2.3.5, 2.3.6, 2.3.7, 2.4.0, 2.4.1, 2.4.3, 2.4.4)
ERROR: No matching distribution found for aiohappyeyeballs==2.6.1
```

ashetkar reviewed (approved):
The steps worked fine on Mac. I'm okay to merge it now and address the centos-related comment later.

nchandrappa committed:
Updating the python pre-req version to 3.11 or later version.
